data_managers/data_manager_build_kraken_database/tool_dependencies.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <set_environment version="1.0">
+        <environment_variable action="set_to" name="KRAKEN_SCRIPT_PATH">$REPOSITORY_INSTALL_DIR</environment_variable>
```

Comment:
I would also not support *needlessly* increasing the minimal Galaxy version (to a point) but tools that use this idiom are pretty much broken if not installed via the tool shed - there are potentially workarounds sure - but I think making the tools easy to use without going through the install process is very important.

You get to pick your kind of broken - broken pre-15.03 or broken if not installed via the Tool Shed forever. I much prefer the first kind of broken.